gene2phenotype_project/gene2phenotype_app/serializers.py

```diff
@@ -69,7 +68,26 @@ def create_stable_id():
 
         return stable_id_instance
     
-    
+    def update_stable_id(self, is_live):
```

Comment:
```suggestion
    def update_stable_id_live_status(self, is_live):
```

Reply:
It makes sense to include 'status' in the name.
Update to `update_g2p_id_status`.